include/rtc/rtpdepacketizer.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2024 Paul-Louis Ageneau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef RTC_RTP_DEPACKETIZER_H
+#define RTC_RTP_DEPACKETIZER_H
+
+#if RTC_ENABLE_MEDIA
+
+#include "mediahandler.hpp"
+#include "message.hpp"
+
+namespace rtc {
+
+/// RTP depacketization for Opus
+class RTC_CPP_EXPORT OpusRtpDepacketizer : public MediaHandler {
```

Comment:
You can actually rename it `AudioRtpDepacketizer` and define depacketizers for both Opus and AAC with:
```
using OpusRtpDepacketizer = AudioRtpDepacketizer;
using AACRtpDepacketizer = AudioRtpDepacketizer;
```
Packetizers are identical too and defined this way.

Reply:
Any reason why you want to prefix Audio while the RtcPacketizer doesn't ?

Reply:
Good question, I guess it could be named `RtpDepacketizer` too. Opus and AAC packetizers are defined as `AudioRtpPacketizer`, so it made sense to do something symmetrical. An `RtpDepacketizer` base class could make sense especially if there is common logic in the future (possibly stuff related to frame info).

Reply:
Good question, I guess it could be named `RtpDepacketizer` too. Opus and AAC packetizers are defined as `AudioRtpPacketizer`, so it made sense to do something symmetrical.
https://github.com/paullouisageneau/libdatachannel/blob/437a758ae89e5342940c139b8a647bf1bb1c0ee5/include/rtc/rtppacketizer.hpp#L61

An `RtpDepacketizer` base class could make sense especially if there is common logic in the future (possibly stuff related to frame info).